File: Find-numbers-with-even-number-of-digits/Find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int res = 0;
        for(int i=0; i < nums.size(); i++)
        {
            if((1 <= log10(nums[i])) && (log10(nums[i]) < 2 )
                || (3 <= log10(nums[i])) && (log10(nums[i]) < 4 )
               || (5 == log10(nums[i])))
                res++;
        }
        return res;
    }
};

File: Find-numbers-with-even-number-of-digits/Find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int result = 0;
        for(int i=0; i < nums.length; i++)
        {
            if (((10 <= nums[i]) && (nums[i] < 100)) 
                ||(((1000 <= nums[i]) && (nums[i] < 10000)))
               || (nums[i] == 100000))
                result++;
        }
        return result;
    }
}
